about.html

```diff
@@ -12,8 +12,25 @@
 </head>
 
 <body>
+
+  <nav>
```

Comment:
Unless all of your nav <a> links are different you do not need to specify a class for each one, the nav and nav a can take care of all of it for you.